### ISSUE TITLE
Unable to revoke certificate

### ISSUE THREAD

**Issue by tenf00t** [CSIS/EnrollmentStation#24]
Hello, we are currently evaluating the YubiKey for use by our users. I installed a new root CA and configured it with the instruction from your documentation. I have been able to use the CSIS Enrollment Station to issue my first certificate for one user and was able to succesfully log into the users workstation via the smartcard. I now want to revoke the certificate to make more tests and I'm getting the following error. Please see screenshot:

https://dl.dropboxusercontent.com/u/7894017/Yubi_Error.jpg

Doing a google search, it seems to be because I'm running the Enrollment Station on a 64bit OS (Windows 10)

I'm using this version of the enrollment station https://github.com/CSIS/EnrollmentStation/releases/download/0.3.5.0/Enrollment.Station.v0.3.5.0.zip

How can I get around this issue?

Thank you

**Comment by mike-csis** [comment]
Hi,

We encountered this issue a little while back and found that a workaround for it.

The issue is that the CertAdm COM library (iirc) isn't installed. We haven't found a way to install just this, but one that does work is described on page 10 of the manual, under troubleshooting. We installed the RSAT tools (remote administration), which amongst others registers the above library. You don't have to connect to anything using the tools.

Hope this helps.
